lib/send_message.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+class SendMessage
+
+  def send_the_message
```

Comment:
Could this method be broken down into more methods.  It feels like it has two responsibilities.